decidim-elections/app/packs/src/decidim/elections/election_log.js

```diff
@@ -5,6 +5,11 @@
 import { Client, MessageParser } from "@decidim/decidim-bulletin_board";
 
 $(async () => {
+  const isSafariBrowser = (/^((?!chrome|android).)*safari/i).test(navigator.userAgent); 
```

Comment:
Trailing space detected by [codeclimate](https://codeclimate.com/github/decidim/decidim/pull/12217) :/  

```suggestion
  const isSafariBrowser = (/^((?!chrome|android).)*safari/i).test(navigator.userAgent);
```